src/Sulu/Bundle/AdminBundle/Admin/SecurityContextAwareInterface.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Admin;
+
+/**
+ * System (string) -> Section (string) -> Context (string) -> Permissions (list of strings).
+ *
+ * @phpstan-type SecurityContextInfo array<string, array<string, array<string, string[]>>>
+ */
+interface SecurityContextAwareInterface
```

Comment:
We don't require a `phpstan-type` here as all our admin classes extends from `Admin` and so a type is not necessary here. We just need define it correctly in the `Admin` class.

Reply:
I used the phpstan type because the functions `getSecurityContext` and `getSecurityContextWithPlaceholders` have the same complex array type and they should have the same structure. Otherwise we would have to define this array shape twice.